src/net.cpp

```diff
@@ -1635,9 +1638,13 @@ void CConnman::ThreadOpenConnections(const std::vector<std::string> connect)
             }
 
             if (add_fixed_seeds_now) {
+                std::vector<CAddress> seed_addr{ConvertSeeds(Params().FixedSeeds())};
+                seed_addr.erase(std::remove_if(seed_addr.begin(), seed_addr.end(),
```

Comment:
Wondering if this deserves a comment? For example, for a confused users which had this working previously.

Reply:
Oops, I meant a Log statement, not a comment.

Reply:
I added a log "Added XXX fixed seeds from reachable networks", that would make it clear, I also think it might be of interest how many addresses were loaded (there is a similar log for the DNS seeds).